short-read-mngs/experimental.wdl

```diff
@@ -219,6 +219,52 @@ task NonhostFastq {
     docker: docker_image_id
   }
 }
+task RunGlueJob {
+  input {
+    String docker_image_id
+    String s3_wd_uri
+  }
+  command<<<
+  # This command is specific to the CZ ID system
+  
+  set -euxo pipefail 
+  BUCKET=$(echo "~{s3_wd_uri}" | cut -d/ -f 3)
+  PIPELINE_RUN_ID=$(echo "~{s3_wd_uri}" | cut -d/ -f 7)
+  if [[ $BUCKET == "idseq-samples-sandbox" ]]; then
+   ENV=sandbox
+  elif [[ $BUCKET == "idseq-samples-staging" ]]; then
+   ENV=staging
+  elif [[ $BUCKET == "idseq-prod-samples-us-west-2" ]]; then
+   ENV=prod
+  else 
+    exit 0
+  fi
+  JOBID=$(aws glue start-job-run --job-name "$ENV"_heatmap_es_job --region us-west-2 --arguments="--user_pipeline_run_ids=$PIPELINE_RUN_ID,--user_background_id=26,--job_type=selected_runs" | jq -r .JobRunId) 
+  i=0
+  while true; do 
+    JOBSTATE=$(aws glue get-job-run --job-name staging_heatmap_es_job --run-id $JOBID | jq -r .JobRun.JobRunState)
+    if [[ $JOBSTATE == "SUCCEEDED" ]]; then 
+      exit 0 
+    elif [[ $JOBSTATE == "FAILED" ]]; then 
```

Comment:
I think that you will have to handle more potential states. https://docs.aws.amazon.com/glue/latest/dg/job-run-statuses.html

Reply:
Thanks!